### ISSUE TITLE
window.OK.callbacks[callbackIndex] is not a function

### ISSUE THREAD

**Issue by kpuzyrenko** [nygardk/react-share#326]
I tried using OKShareCount and i get the error "window.OK.callbacks[callbackIndex] is not a function"

Here is how I use it: 
```
import {OKShareCount} from 'react-share';

class Share extends Component {
render() {
		return (
			<OKShareCount url={path}>
				<span className={styles.shareCount}>{shareCount}</span>}
			</ShareCount>
		);
	}
}
export default Share; 
```

The site is not in prod yet though 
Do you think you can fix?
Thank you

**Comment by nygardk** [comment]
Hello! This should be fixed in `v4.2.0`.

**Comment by kpuzyrenko** [comment]
Thank you very much!
